contributionguidelines.md

```diff
@@ -0,0 +1,21 @@
+# How to Contribute
```

Comment:
Convention is to have this file be named `CONTRIBUTING.md`. I'd like to see this file renamed to match https://github.com/codeconnector/codeconnector.github.io/blob/master/CONTRIBUTING.md

Reply:
Understood will change the filename to match.